unit_tests/CMakeLists.txt

```diff
@@ -328,17 +393,31 @@ if(WIN32)
         # Have CMake invoke itself to performa a local install for our test suite.
         #
         if(ENABLE_APP)
-            add_custom_target(test_install
```

Comment:
Instead of this, we can also simplify by using `add_dependencies()` for the `test_install` target after we add the custom target. Eg.
```cmake
add_custom_target(test_install
    ALL
    "${CMAKE_COMMAND}"
    -D CMAKE_INSTALL_CONFIG_NAME:string=$<CONFIG>
    -D CMAKE_INSTALL_PREFIX:string=$<TARGET_FILE_DIR:check_clamav>
    -P "${CMAKE_BINARY_DIR}/cmake_install.cmake"
    DEPENDS
        check_clamav check_clamd check_fpu_endian
        ClamAV::libclamav ClamAV::libfreshclam ClamAV::libunrar ClamAV::libunrar_iface 
        clambc clamd clamdscan clamdtop clamscan clamsubmit clamconf freshclam-bin sigtool
)

if(ENABLE_EXTERNAL_MSPACK)
    add_dependencies(test_install MSPack::mspack)
else()
    add_dependencies(test_install ClamAV::libmspack)
endif()
```